eng/packages/http-client-csharp/generator/Azure.Generator/src/Primitives/NewAzureProjectScaffolding.cs

```diff
@@ -37,9 +37,10 @@ protected override string GetSourceProjectFileContent()
             }
 
             int pathSegmentCount = GetPathSegmentCount();
+            const string corePathSegment = "Shared/Core";
```

Comment:
nit: can we move this to where we define the other constants in this file?
Also, name it something like `SharedSourceLinkBase`?